get-started/using/index.md

```diff
@@ -59,11 +59,11 @@ title: Using Monero
                 <div class="info-block text-adapt">
                     <div class="row center-xs">
                         <div class="col">
-                            <h2>4. Getting Monero</h2>
+                            <h2>Obtain Monero</h2>
                         </div>
                     </div>
                     <div class="row start-xs">
-                        <p>Empty wallets are no fun! Let’s get some Monero in there! You can receive Monero from someone else, or purchase some on <a href="/community/merchants/">an exchange</a>. We do not endorse specific services, but you can view the exchanges that support Monero on the Merchants page. Depending on the exchange, you can buy Monero from several currencies including USD, EUR, and Bitcoin. For price speculation and talk, join the active trading discussion at <a href="irc://chat.freenode.net/#monero-markets">#monero-markets.</a></p>
+                        <p>One may purchase Monero on an <a href="/community/merchants/">exchange</a> with fiat or other cryptocurrencies. 
```

Comment:
> One may

Although `you` and `one` are both considered impersonal/indefinite pronouns we should probably avoid mixing them. Generally `you` is considered more informal/friendly in American English. Also,  `may` connotes permission where `can` permits capability to do so. Perhaps: `You can purchase`.

Also, this might be a great spot to insert something about mining.